src/io/flutter/run/OpenDevToolsAction.java

```diff
@@ -72,6 +73,17 @@ public void actionPerformed(@NotNull final AnActionEvent event) {
       return;
     }
 
+    // This action is registered in plugin.xml with the default constructor.
+    // Therefore, if a user triggers this from the IDE, even if there is a
+    // running Flutter app myConnector will be null. In that case, check for a
+    // Flutter app first and use its connector instead.
+    if (myConnector == null) {
+      final List<FlutterApp> apps = FlutterApp.allFromProjectProcess(project);
+      if (!apps.isEmpty()) {
+        myConnector = apps.get(0).getConnector();
```

Comment:
Is it possible to find the currently activated app instead of always using the first?

Reply:
I couldn't figure out a way to do that unfortunately 😮‍💨 I added a TODO though to capture that.